### PR TITLE
DOC-1542: Bespoke Names Changes

### PR DESCRIPTION
**Pull request by thedrjim**
Related Ticket: DOC-1542

Description of Changes:
* Changed updated setting names to there 6v conterparts 

Pre-checks: DOC-1542
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/font_size_formats.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[fontsize_formats]]
 == `+fontsize_formats+`
```
Comment (ltrouton) [inline modules/ROOT/partials/configuration/font_size_formats.adoc:2]:
```suggestion
[[font_size_formats]]
== `+font_size_formats+`
```
There also looks to be a few others in the files which haven't been converted

File: modules/ROOT/partials/configuration/style_formats.adoc [hunk 2]
```diff
@@ -74,7 +74,7 @@ tinymce.init({
 This example shows you how to:
 
 * Override the built-in xref:content-formatting.adoc#formats[formats].
-* Add some custom styles to the `+styleselect+` dropdown toolbar button and the `+formats+` menu item using the xref:user-formatting-options.adoc#style_formats[style_formats] configuration option.
+* Add some custom styles to the `+styles+` dropdown toolbar button and the `+formats+` menu item using the xref:user-formatting-options.adoc#style_formats[style_formats] configuration option.
```
Comment (ltrouton) [inline modules/ROOT/partials/configuration/style_formats.adoc:77]: I belive this needs to be changed as well
```suggestion
* Add some custom styles to the `+styles+` dropdown toolbar button and the `+styles+` menu item using the xref:user-formatting-options.adoc#style_formats[style_formats] configuration option.
```

**Review by ltrouton — COMMENTED** [review]
Just to note, it looks like the branch is missing the `feature` prefix so should have been `feature/DOC-1542`.

Also, there are several files that need to be updated since a few of the file names have changed. E.g. `include::partial$configuration/font_formats.adoc[]` needs to be changed to `include::partial$configuration/font_family_formats.adoc[]` so it is properly included

**Comment by lnewson** [comment]
I'm converting this back to a draft as I know @thedrjim is working on the includes, xrefs, etc... we should convert it back when that's done to save others reviewing this when it's not really ready yet.

**Review by EkimChau — APPROVED** [review]

File: modules/ROOT/partials/toolbar-button-ids/core-toolbar-buttons.adoc [hunk 3]
```diff
@@ -11,10 +11,10 @@
 |`+bold+` |Applies the bold format to the current selection.
 |`+copy+` |Copies the current selection into clipboard.
 |`+cut+` |Cuts the current selection into clipboard.
-|`+fontselect+` |Dropdown list with font families to apply to selection.
-|`+fontsizeselect+` |Dropdown list with font sizes to apply to selection.
+|`+fontfamily+` |Dropdown list with font families to apply to selection.
+|`+fontsize+` |Dropdown list with font sizes to apply to selection.
 |`+forecolor+` |Applies foreground/text color to selection.
-|`+formatselect+` |Dropdown list with block formats to apply to selection.
+|`+blocks+` |Dropdown list with block formats to apply to selection.
```
Comment (lnewson) [inline modules/ROOT/partials/toolbar-button-ids/core-toolbar-buttons.adoc:17]: This probably needs move up before `bold`, as this list is alphabetical.

File: modules/ROOT/examples/live-demos/web-component/example.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
       plugins="advlist autolink lists link image charmap preview anchor
       searchreplace visualblocks code fullscreen
       insertdatetime media table paste code help wordcount"
-      toolbar="undo redo | formatselect | bold italic backcolor |
+      toolbar="undo redo | fontfamily | bold italic backcolor |
```
Comment (lnewson) [inline modules/ROOT/examples/live-demos/web-component/example.html:19]:
```suggestion
      toolbar="undo redo | blocks | bold italic backcolor |
```

File: modules/ROOT/examples/live-demos/web-component/index.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
       plugins="advlist autolink lists link image charmap preview anchor
       searchreplace visualblocks code fullscreen
       insertdatetime media table paste code help wordcount"
-      toolbar="undo redo | formatselect | bold italic backcolor |
+      toolbar="undo redo | fontfamily | bold italic backcolor |
```
Comment (lnewson) [inline modules/ROOT/examples/live-demos/web-component/index.html:11]:
```suggestion
      toolbar="undo redo | blocks | bold italic backcolor |
```

File: modules/ROOT/partials/configuration/style_formats.adoc [hunk 6]
```diff
@@ -1,7 +1,7 @@
 [[style_formats]]
 == style_formats
 
-This option allows you to define custom items for the `+styleselect+` dropdown toolbar button and the `+formats+` menu item.
+This option allows you to define custom items for the `+styles+` dropdown toolbar button and the `+formats+` menu item.
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/style_formats.adoc:4]:
```suggestion
This option allows you to define custom items for the `+styles+` dropdown toolbar button and the `+styles+` menu item.
```

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/examples/live-demos/checklist/index.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ tinymce.init({
   menu: {
     format: {
       title: 'Format',
-      items: 'checklist | bold italic underline strikethrough superscript subscript codeformat | formats blockformats fontformats fontsizes align | forecolor backcolor | removeformat'
+      items: 'checklist | bold italic underline strikethrough superscript subscript codeformat | style blocks fontfamily fontsize align | forecolor backcolor | removeformat'
```
Comment (ltrouton) [inline modules/ROOT/examples/live-demos/checklist/index.js:9]:
```suggestion
      items: 'checklist | bold italic underline strikethrough superscript subscript codeformat | styles blocks fontfamily fontsize align | forecolor backcolor | removeformat'
```

File: modules/ROOT/partials/configuration/defaultmenuitems.adoc [hunk 8]
```diff
@@ -7,7 +7,7 @@ tinymce.init({
     edit: { title: 'Edit', items: 'undo redo | cut copy paste | selectall | searchreplace' },
     view: { title: 'View', items: 'code | visualaid visualchars visualblocks | spellchecker | preview fullscreen' },
     insert: { title: 'Insert', items: 'image link media template codesample inserttable | charmap emoticons hr | pagebreak nonbreaking anchor | insertdatetime' },
-    format: { title: 'Format', items: 'bold italic underline strikethrough superscript subscript codeformat | formats blockformats fontformats fontsizes align lineheight | forecolor backcolor | removeformat' },
+    format: { title: 'Format', items: 'bold italic underline strikethrough superscript subscript codeformat | style blocks fontfamily fontsize align lineheight | forecolor backcolor | removeformat' },
```
Comment (ltrouton) [inline modules/ROOT/partials/configuration/defaultmenuitems.adoc:10]:
```suggestion
    format: { title: 'Format', items: 'bold italic underline strikethrough superscript subscript codeformat | styles blocks fontfamily fontsize align lineheight | forecolor backcolor | removeformat' },
```

File: modules/ROOT/partials/integrations/laravel-quick-start.adoc [hunk 9]
```diff
@@ -120,7 +120,7 @@ _File:_ `+resources/views/components/head/tinymce-config.blade.php+`
   tinymce.init({
     selector: 'textarea#myeditorinstance', // Replace this CSS selector to match the placeholder element for TinyMCE
     plugins: 'code table lists',
-    toolbar: 'undo redo | formatselect| bold italic | alignleft aligncenter alignright | indent outdent | bullist numlist | code | table'
+    toolbar: 'undo redo | blocks| bold italic | alignleft aligncenter alignright | indent outdent | bullist numlist | code | table'
```
Comment (ltrouton) [inline modules/ROOT/partials/integrations/laravel-quick-start.adoc:123]:
```suggestion
    toolbar: 'undo redo | blocks | bold italic | alignleft aligncenter alignright | indent outdent | bullist numlist | code | table'
```

**Review by ltrouton — APPROVED** [review]

**Review by tiny-james — APPROVED** [review]

File: modules/ROOT/partials/menu-item-ids/core-menu-items.adoc [hunk 10]
```diff
@@ -3,15 +3,15 @@
...
 |`+align+` |Format |Changes alignment to the current block or selection.
 |`+backcolor+` |Format |Applies background color to selection.
-|`+blockformats+` |Format |Applies block formats to current selection.
+|`+blocks+` |Format |Applies block formats to current selection.
 |`+bold+` |Format |Applies bold format to current selection.
 |`+codeformat+` |Format |Applies inline code format to current selection.
 |`+copy+` |Edit |Copies the current selection into clipboard.
 |`+cut+` |Edit |Cuts the current selection into clipboard.
 |`+forecolor+` |Format |Applies foreground/text color to selection.
 |`+formats+` |Format |Menu of all available formats.
```
Comment (metricjs) [inline modules/ROOT/partials/menu-item-ids/core-menu-items.adoc:12]: This was also renamed to `styles` I believe? Will need to be renamed and moved, to keep alphabetical order. And this description isn't great... We could steal the description from the toolbars list? `Dropdown list with styles to apply to selection.`

**Review by metricjs — CHANGES_REQUESTED** [review]

**Review by metricjs — APPROVED** [review]

**Pull request merged**